src/wp-includes/php-compat/readonly.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Conditionally declares a `readonly()` function, which was renamed
+ * to `wp_readonly()` in WordPress 5.9.0.
+ *
+ * In order to avoid PHP parser errors, this function was extracted
+ * to this separate file and is only included conditionally on PHP 8.1.
+ *
+ * Including this file on PHP >= 8.1 results in a fatal error.
+ *
+ * @package WordPress
+ * @since 5.9.0
```

Comment:
Shouldn't this have "wp-5.9.0"?

Reply:
It should - and in some other placed in their file too - updated PR to follow.